superset-frontend/src/components/Menu/Menu.tsx

```diff
@@ -195,8 +196,15 @@ export function Menu({
                 {!navbarRight.user_is_anonymous && [
                   <DropdownMenu.Divider key="user-divider" />,
                   <DropdownMenu.ItemGroup key="user-section" title={t('User')}>
-                    <DropdownMenu.Item key="profile">
-                      <a href={navbarRight.user_info_url}>{t('Profile')}</a>
+                    {navbarRight.user_profile_url && (
+                      <DropdownMenu.Item key="profile">
+                        <a href={navbarRight.user_profile_url}>
+                          {t('Profile')}
+                        </a>
+                      </DropdownMenu.Item>
+                    )}
+                    <DropdownMenu.Item key="edit">
+                      <a href={navbarRight.user_info_url}>{t('Edit')}</a>
```

Comment:
The default user info url doesn't really allow users to edit their profile. Maybe we can just remove this link for now?

Reply:
Really? When would a user ever not be able to change their name?

Reply:
After looking around, it seems like maybe only admins can edit users.

So I changed the text here to say "Info" instead, which seems to match the url and view name too, so 🤷